test/unit/rules/functions/test_sub_needed.py

```diff
@@ -27,13 +27,13 @@ def test_template_config(self):
         """Test custom excludes configuration"""
         self.helper_file_rule_config(
             'test/fixtures/templates/good/functions/sub_needed_custom_excludes.yaml',
-            {'custom_excludes': '^\\$\\{self.+\\}$'}, 0
+            {'custom_excludes': '^\\$\\{Stage\\}$'}, 0
```

Comment:
wonder if `custom_excludes` are even needed anymore

although we don't want to break the interface yet and should allow them to be passed in for backwards compatibility